src/sonic-config-engine/minigraph.py

```diff
@@ -2040,6 +2040,9 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     results['ACL_TABLE'] = filter_acl_table_bindings(acls, neighbors, pcs, pc_members, sub_role, current_device['type'] if current_device else None, is_storage_device, vlan_members)
     results['FEATURE'] = {
         'telemetry': {
```

Comment:
>telemetry [](http://example.com/codeflow?start=9&length=9)

If build time flag `INCLUDE_SYSTEM_TELEMETRY` already disabled, we should not allow runtime feature enable/disable at all.

Reply:
INCLUDE_SYSTEM_TELEMETRY is used to build docker image for telemetry, but now we don't have docker-sonic-telemetry.